segfault in latest release on `envelop` command

Working through the steps in the [docs](https://docs.gobl.org/quick-start/invoices), there appears to be a segfault when trying to envelop.

Command:

```
gobl envelop -t bill.Invoice -i invoice.yaml
```

Output:

```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x2 addr=0x2 pc=0x10539e440]

goroutine 1 [running]:
main.(*rootOpts).openOutput(0x0, 0x1400032ca00, {0x140002dfe40, 0x1, 0x4})
	/Users/sam/go/pkg/mod/github.com/invopop/gobl.cli@v0.26.0/cmd/gobl/root.go:66 +0x20
main.(*buildOpts).runE(0x140000cff80, 0x1400032ca00, {0x140002dfe40, 0x1, 0x4})
	/Users/sam/go/pkg/mod/github.com/invopop/gobl.cli@v0.26.0/cmd/gobl/build.go:92 +0x18c
github.com/spf13/cobra.(*Command).execute(0x1400032ca00, {0x140002dfe00, 0x4, 0x4})
	/Users/sam/go/pkg/mod/github.com/spf13/cobra@v1.4.0/command.go:856 +0x668
github.com/spf13/cobra.(*Command).ExecuteC(0x1400032c500)
	/Users/sam/go/pkg/mod/github.com/spf13/cobra@v1.4.0/command.go:974 +0x410
github.com/spf13/cobra.(*Command).Execute(...)
	/Users/sam/go/pkg/mod/github.com/spf13/cobra@v1.4.0/command.go:902
github.com/spf13/cobra.(*Command).ExecuteContext(...)
	/Users/sam/go/pkg/mod/github.com/spf13/cobra@v1.4.0/command.go:895
main.run()
	/Users/sam/go/pkg/mod/github.com/invopop/gobl.cli@v0.26.0/cmd/gobl/main.go:43 +0x118
main.main()
	/Users/sam/go/pkg/mod/github.com/invopop/gobl.cli@v0.26.0/cmd/gobl/main.go:20 +0x20
```

--- a/cmd/gobl/build.go
+++ b/cmd/gobl/build.go
@@ -36,11 +36,12 @@ func build(root *rootOpts) *buildOpts {
 	}
 }
 
-func envelop() *buildOpts {
+func envelop(root *rootOpts) *buildOpts {
 	return &buildOpts{
-		envelop: true,
-		use:     "envelop [infile] [outfile]",
-		short:   "Prepare a document and insert into a new envelope",
+		rootOpts: root,
+		envelop:  true,
+		use:      "envelop [infile] [outfile]",
+		short:    "Prepare a document and insert into a new envelope",
 	}
 }
 

--- a/cmd/gobl/root.go
+++ b/cmd/gobl/root.go
@@ -20,21 +20,20 @@ func root() *rootOpts {
 }
 
 func (o *rootOpts) cmd() *cobra.Command {
-	opts := &rootOpts{}
 	cmd := &cobra.Command{
 		Use:           "gobl",
 		SilenceUsage:  true,
 		SilenceErrors: true,
 	}
 
-	opts.setFlags(cmd)
+	o.setFlags(cmd)
 
 	cmd.AddCommand(verify().cmd())
-	cmd.AddCommand(envelop().cmd()) // see build
-	cmd.AddCommand(build(opts).cmd())
+	cmd.AddCommand(envelop(o).cmd()) // see build
+	cmd.AddCommand(build(o).cmd())
 	cmd.AddCommand(version())
 	cmd.AddCommand(serve().cmd())
-	cmd.AddCommand(keygen(opts).cmd())
+	cmd.AddCommand(keygen(o).cmd())
 	return cmd
 }
 

--- a/samples/invoice-es.yaml
+++ b/samples/invoice-es.yaml
@@ -1,0 +1,35 @@
+currency: "EUR"
+issue_date: "2022-02-01"
+code: "SAMPLE-001"
+
+supplier:
+  tax_id:
+    country: "ES"
+    code: "B98602642" # random
+  name: "Provider One S.L."
+  emails:
+    - addr: "billing@example.com"
+  addresses:
+    - num: "42"
+      street: "Calle Pradillo"
+      locality: "Madrid"
+      region: "Madrid"
+      code: "28002"
+      country: "ES"
+
+customer:
+  tax_id:
+    country: "ES"
+    code: "54387763P"
+  name: "Sample Consumer"
+
+lines:
+  - quantity: 10
+    item:
+      name: "Item being purchased"
+      price: "100.00"
+    discounts:
+      - percent: "10%"
+    taxes:
+      - cat: "VAT"
+        rate: "standard"

